plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -56,7 +56,10 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "QSV_Encoder_Internal.h"
 #include "QSV_Encoder.h"
-#include <mfxvideo++.h>
+#include <vpl/mfxstructures.h>
+#include <vpl/mfxvideo++.h>
+#include <vpl/mfxdispatcher.h>
+#include <VersionHelpers.h>
```

Comment:
`VersionHelpers.h` doesnt exist was it supposed to be added as part of these changes or was it an include that is no longer used?